.circleci/scripts/binary_upload.sh

```diff
@@ -24,6 +24,8 @@ if [[ "${DRY_RUN}" = "disabled" ]]; then
   AWS_S3_CP="aws s3 cp"
 fi
 
+retry () { "$@"  || (sleep 1 && "$@") || (sleep 2 && "$@") }
```

Comment:
Why not source it from that script (rather than copy-n-paste the implementation?)

Reply:
Using source import I get this error:
```
Run bash .circleci/scripts/binary_upload.sh
.circleci/scripts/../../.jenkins/pytorch/common.sh: line 12: BUILD_ENVIRONMENT: unbound variable
Error: Process completed with exit code 1.
```